internal/symbolz/symbolz.go

```diff
@@ -63,10 +63,29 @@ func Symbolize(p *profile.Profile, force bool, sources plugin.MappingSources, sy
 	return nil
 }
 
+// Check whether path ends with one of the suffixes listed in
+// pprof_remote_servers.html from the gperftools distribution
+func hasLegacySuffix(path string) bool {
```

Comment:
I am not sure "legacy" is the best name. The remote serving is a feature supported by gperftools, right? Is that really "legacy"? Maybe it should be called "hasGperftoolsSuffix"?

Reply:
The gperftools devs themselves call their version of pprof legacy and recommend switching to this one. Yet, `hasGperftoolsSuffix` would certainly be clearer.